apps/docusaurus/docs/designsystems/Komponenter/checkbox.md

```diff
@@ -36,18 +39,122 @@ Avmerkingsboksen får fokus når den velges med tastaturet.
 
 ## Feilmelding avmerkingsboks
 
-Følgende avmerkingsboks får feilmeldingsfarge hvis ett av alternativene må være valgt før bruker kan gå videre.
+Følgende avmerkingsboks får feilmeldingsfarge og en feilmelding hvis ett av alternativene må være valgt før bruker kan
+gå videre.
 
-<div class="component__display">
+<div class="component__display__column">
     <div class="kv-checkbox">
         <input type="checkbox" id="checkbox_2" required/>
-        <label class="detail detail--sml" for="checkbox_2">Input tekst</label>
+        <label class="detail detail--sml" for="checkbox_2">Jeg aksepterer vilkårene for...</label>
+        <div class="error__box">
+            <i class="material-symbols-outlined">exclamation</i>
+            <small>Du må huke av dette feltet</small>
+        </div>
     </div>
 </div>
+<br/>
+<br/>
 
 ```markdown
 <div class="kv-checkbox">
-    <input type="checkbox" id="checkbox_2" required/>
-    <label class="detail detail--sml" for="checkbox_2">Input tekst</label>
+    <input type="checkbox" id="checkbox_6" required/>
+    <label class="detail detail--sml" for="checkbox_6">Jeg aksepterer vilkårene for...</label>
+    <div class="error__box">
+        <i class="material-symbols-outlined">exclamation</i>
+        <small>Du må huke av dette feltet</small>
+    </div>
+</div>
+```
+
+## Eksempler
+
+### Eksempel for feilmelding
+
+Marker hvilket felt som MÅ avhukes
+
+<div class="component__display__column">
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_3"/>
+        <label class="detail detail--sml" for="checkbox_3">Jeg ønsker å få tilsendt nyhetsbrevet</label>
+    </div>
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_4" required/>
+        <label class="detail detail--sml" for="checkbox_4">Jeg aksepterer vilkårene</label>
+        <div class="error__box">
+        <i class="material-symbols-outlined">exclamation</i>
+        <small>Du må huke av dette feltet</small>
+        </div>
+    </div>
+</div>
+
+<br/>
+<br/>
+
+```markdown
+<div class="kv-checkbox">
+    <input type="checkbox" id="checkbox_3"/>
+    <label class="detail detail--sml" for="checkbox_3">Jeg ønsker å få tilsendt nyhetsbrevet</label>
+</div>
+<div class="kv-checkbox">
+    <input type="checkbox" id="checkbox_4"/>
+    <label class="detail detail--sml" for="checkbox_4">Jeg aksepterer vilkårene</label>
+    <div class="error__box">
+        <i class="material-symbols-outlined">exclamation</i>
+        <small>Du må huke av dette feltet</small>
+    </div>
 </div>
 ```
+
+### Eksempel for "velg minst én"
+
+"Velg minst Én" er forskjellig fra radioknapper, som lar deg bare velge en.
+
+Hvordan kommer du deg til jobb?
+
+<div class="component__display__column">
```

Comment:
Denne klassen er en del av custom-css for docusaurus, ikke en del av selve KVIB-biblioteket. Den er derfor ikke tilgjengelig for brukerne våre, så da tror jeg ikke den bør være en del av eksempelet. Jeg er litt usikker på hva vi skal gjør i stedenfor her. Å lage en egen container-klasse for checkboxer kan være et alternativ, men vi bør teste litt og snakke med designerene før vi bestemmer oss.